src/Main.java

```diff
@@ -1,5 +1,78 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Random;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+    private static final Employee[] employees = {
+            new Employee("Rostkova Pipa Pipachkina", 999_000, 3),
+            new Employee("Zanina Nina Ninachkina", 800_000, 2),
+            new Employee("Pusikovich Alex Alexandrovich", 5_000, 4),
+            new Employee("Popkov Tolik Tolikovich", 950_000, 5),
+            new Employee("Bublik Bastard Bastardovich", 10_000, 1)
+    };
+    }
+
+    public static void printAllEmployees() {
+        System.out.println("Список сотрудников:");
+        for (Employee employee : employees) {
+            if (employee != null) {
+                System.out.println(employee);
+            }
+        }
+    }
+
+    private static int calculateSumOfSalaries() {
+        int sum = 0;
+        for (int i = 0; i < employees.length; i++) {
+            Employee employee = employees[i];
+            if (employee != null) {
+                sum += employee.getSalaryEmployee();
+            }
+        }
+        return sum;
+    }
+
+    private static Employee findEmployeeWithMinSalary() {
+        Employee employeeWithMinSalary = null;
```

Comment:
Сразу присвой первое значение, и тогда ниже на нулл не придется проверять
```suggestion
        Employee employeeWithMinSalary = employess[0];
```